gammapy/datasets/tests/test_evaluator.py

```diff
@@ -31,8 +31,8 @@ def evaluator():
     )
 
     spectral_model = ConstantSpectralModel()
-    spatial_model = PointSpatialModel(
-        lon_0=0 * u.deg, lat_0=0 * u.deg, frame="galactic"
+    spatial_model = GaussianSpatialModel(
```

Comment:
I propose to keep the test with the PointSpatialModel... We had issues in the past with such model

Reply:
done